### PR TITLE
add odbc CI

### PR DESCRIPTION
**Pull request by borune-k12**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by borune-k12** [comment]
Rerun check after #73 will be merged

File: .github/workflows/odbc.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: odbc CI
+
+on:
+  pull_request:
+    paths:
+      - 'recipes/odbc/**'
```
Comment (Artalus) [inline .github/workflows/odbc.yml]:
```suggestion
      - 'recipes/odbc/all/**'
```

File: .github/workflows/odbc.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      - 'build.py'
+      - '.github/workflows/odbc.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - 'recipes/odbc/**'
```
Comment (Artalus) [inline .github/workflows/odbc.yml]:
```suggestion
      - 'recipes/odbc/all/**'
```

File: .github/workflows/odbc.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+env:
+  CONAN_PASSWORD: ${{ secrets.BintrayApiKey }}
+  IS_PURE_C: true
+
+jobs:
+  odbc:
+    strategy:
+      matrix:
+        build: ['linux-gcc4.8', 'linux-gcc7', 'linux-gcc9', 'windows-2019-release', 'windows-2019-debug', 'macos']
```
Comment (Artalus) [inline .github/workflows/odbc.yml]: оно же вроде не собирается на винде согласно https://github.com/trassir/conan-center-index/blob/master/recipes/odbc/all/conanfile.py#L25

**Pull request merged**